src/applications/financial-status-report/components/household/DependentAges.jsx

```diff
@@ -178,7 +178,7 @@ const DependentAges = ({
         name={`dependentAge-${i}`}
         onInput={({ target }) => handlers.updateDependents(target, i)}
         value={dependent.dependentAge}
-        className="input-size-2 no-wrap"
+        width="md"
         onBlur={event => handlers.handleBlur(event, i)}
```

Comment:
We should double check mobile when this gets to stage

Reply:
Yup, I checked mobile. Using the `width="md"` prop fixes the no-wrap issues we were having.